tests/unit/api/bucketGetLifecycle.js

```diff
@@ -0,0 +1,102 @@
+const assert = require('assert');
+
+const { bucketPut } = require('../../../lib/api/bucketPut');
+const bucketGetLifecycle = require('../../../lib/api/bucketGetLifecycle');
+const bucketPutLifecycle = require('../../../lib/api/bucketPutLifecycle');
+const { cleanup,
+    DummyRequestLogger,
+    makeAuthInfo }
+    = require('../helpers');
+
+const log = new DummyRequestLogger();
+const authInfo = makeAuthInfo('accessKey1');
+const bucketName = 'bucketname';
+const testBucketPutRequest = {
+    bucketName,
+    headers: { host: `${bucketName}.s3.amazonaws.com` },
+    url: '/',
+};
+
+function _makeLifecycleRequest(xml) {
```

Comment:
Looks like these private test methods are duplicates of the utility methods in tests/unit/utils/lifecycleHelpers.js. (Sorry in advance if I'm reviewing this too soon 😸.)

Reply:
Yeah, they basically are. Next step was going to be to pull them out :)